src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -111,17 +112,31 @@ class BookingFacade(
     }
   }
 
-  private fun sendProbationBookingEmails(action: BookingAction, booking: VideoBooking, prisoner: Prisoner, user: User?) {
-    log.info("TODO - send probation booking emails.")
+  private fun sendProbationBookingEmails(eventType: BookingAction, booking: VideoBooking, prisoner: Prisoner, user: User?) {
+    val appointment = booking.appointments().single()
+    val prison = prisonRepository.findByCode(prisoner.prisonCode)!!
+    val contacts = contactsService.getPrimaryBookingContacts(booking.videoBookingId, user).allContactsWithAnEmailAddress()
+    val location = locationsInsidePrisonClient.getLocationByKey(appointment.prisonLocKey)!!
+
+    contacts.mapNotNull { contact ->
+      when (contact.contactType) {
+        ContactType.USER -> ProbationEmailFactory.user(contact, prisoner, booking, prison, appointment, location, eventType)
```

Comment:
This the initial step towards adding more emails.  We only deal with the user at the moment.